H4LCppModule.py

```diff
@@ -59,31 +63,54 @@ def beginJob(self):
         pass
 
     def endJob(self):
-        print("PassTrig: "+str(self.passtrigEvts)+" Events")
-        print("Pass4eCut: "+str(self.worker.cut4e)+" Events")
+        print("\n========== Print Cut flow table  ====================\n")
+        print("{:27}:{:7} {}".format("Total: ", str(self.noCutsEvts), " Events"))
+        print("{:27}:{:7} {}".format("PassTrig: ", str(self.passtrigEvts), " Events"))
+        print("{:27}:{:7} {}".format("Pass4eCut: ", str(self.worker.cut4e), " Events"))
         print("Pass4eGhostRemoval: "+str(self.worker.cutghost4e)+" Events")
         print("Pass4eLepPtCut: "+str(self.worker.cutLepPt4e)+" Events")
         print("Pass4eQCDSupress: "+str(self.worker.cutQCD4e)+" Events")
```

Comment:
Update the above three lines similar pattern what we used for all others. I mean like below

```python
print("{:27}:{:7} {}".format("Pass4eCut: ", str(self.worker.cut4e), " Events"))
```